packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionHierarchy_def.hpp

```diff
@@ -1568,8 +1582,11 @@ computeResidual(std::vector<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdin
     )
 {
 #include "Xpetra_UseShortNames.hpp"
+  using Teuchos::TimeMonitor;
   const int maxRegPerProc = regX.size();
 
+  RCP<TimeMonitor> tm = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("computeResidual: 1 - Rreg = Areg*Xreg")));
```

Comment:
Did I mention how I feel about handling a scope guard by pointer?  Oh wait I did ;-)

Reply:
Sorry, I was dumbly copy pasting a pattern used throughout the muelu examples...
I will try to keep this in mind and change the pattern next time I need it : )

Reply:
Thanks @lucbv ! :-D  If using a scope guard in the proper way is not convenient, you could always just call `start` and `stop` on the timer.